axolotl-web/src/App.vue

```diff
@@ -1,56 +1,56 @@
 <template>
   <div id="app" @click="checkDebug">
-    <header-comp></header-comp>
+    <header-comp />
     <main class="container">
-      <error-modal  v-if="error"/>
+      <error-modal v-if="error" />
       <router-view />
     </main>
   </div>
 </template>
 
 <script>
-window.getCookie = function(cname) {
-      var name = cname + "=";
-      var ca = document.cookie.split(';');
-      for(var i = 0; i < ca.length; i++) {
-        var c = ca[i];
-        while (c.charAt(0) == ' ') {
-          c = c.substring(1);
-        }
-        if (c.indexOf(name) == 0) {
-          return c.substring(name.length, c.length);
-        }
-      }
-      return false;
-}
-if (window.getCookie("darkMode") === 'true') {
-  import ('./assets/dark.scss');
+window.getCookie = function (cname) {
+  var name = cname + "=";
+  var ca = document.cookie.split(";");
+  for (var i = 0; i < ca.length; i++) {
+    var c = ca[i];
+    while (c.charAt(0) == " ") {
```

Comment:
I cannot say completely understand the ins and outs of JavaScript equality checks, but I tend to use `===` whenever possible. I am sure we are probably fine keeping `==`, but staying safe might save us in the long run?

Both here as well as on line 20 below, I therefor suggest to use the safe triple equality check unless, and if we need the double equality, lets maybe document why?

Reply:
double equals (==) will perform a type conversion when comparing two things, and will handle NaN, -0, and +0 specially to conform to IEEE 754 (so NaN != NaN, and -0 == +0);


triple equals (===) will do the same comparison as double equals (including the special handling for NaN, -0, and +0) but without type conversion; if the types differ, false is returned.double equals (==) will perform a type conversion when comparing two things, and will handle NaN, -0, and +0 specially to conform to IEEE 754 (so NaN != NaN, and -0 == +0);


triple equals (===) will do the same comparison as double equals (including the special handling for NaN, -0, and +0) but without type conversion; if the types differ, false is returned.

so here we can use `===`